src/main/kotlin/sidechain/eth/EthChainListener.kt

```diff
@@ -32,7 +32,7 @@ class EthChainListener(
     override fun getBlockObservable(): Result<Observable<EthBlock>, Exception> {
         return Result.of {
             // convert rx1 to rx2
-            RxJavaInterop.toV2Observable(web3.blockObservable(false))
+            web3.blockFlowable(false).toObservable()
```

Comment:
Please, remove comment above and remove RxJavaInterop and RxJava1 from gradle dependencies.